### ISSUE TITLE
IDE doesn't quit on macOS

### ISSUE THREAD

**Issue by dyfer** [supercollider/supercollider#4300]
## Environment

* SuperCollider version: develop 0a601d50c
* Operating system: macOS 10.13.6
* Other details (Qt version, audio driver, etc.): QT 5.12.0 (homebrew), build config RelWithDebInfo, Debug

## Steps to reproduce
Open SuperCollider (IDE).
Quit the IDE (either via top menu or cmd-q)

## Expected vs. actual behavior

Expected: sclang quits, IDE closes (quits) as well.

Actual: sclang quits, but the IDE stays open. _Cmd-q doesn't work._

The behavior differs slightly whether the attempt to close SC was done using the keyboard shortcut or the top menu:
- if it was done using the top menu, then after the interpreter quits the menu still has the option to quit SuperCollider. The menu still works and can quit the IDE (on the second attempt)
- if it was done using cmd-q, then there is no option in the menu to quit the IDE. In that case the IDE can be quit by pressing the close window icon ("x") in the top left corner.

(no) error message after attempting to close the IDE:
```
main: waiting for input thread to join...
main: quitting...
cleaning up OSC
Interpreter has quit. [Exit code: 0]
```
After this the IDE stays open.

**Comment by brianlheim** [comment]
Can't reproduce. Please try on a clean build. Are you building release or debug?

**Comment by dyfer** [comment]
I'm building release with debug symbols. 
I tried:
- clean build
- removing user support dir (settings, quarks, etc)
- debug build
and it's the same result... Maybe it's qt version?

**Comment by brianlheim** [comment]
which target are you building and running?

**Comment by dyfer** [comment]
> which target are you building and running?

`install` I think:
`cmake --build . --target install --config RelWithDebInfo`

**Comment by michaeldzjap** [comment]
I'm experiencing the same on _macOS_ 10.14.3. Also, when I start the IDE I just get a black screen. I have to manually resize the IDE window in order to see any content. I'm also using _Qt_ v5.12.0 (via _Homebrew_) and build using the same command as @dyfer.

**Comment by snappizz** [comment]
fixed in #4500, thanks @geoffroymontel!
